Meshtastic/Helpers/BLEManager.swift

```diff
@@ -898,8 +906,8 @@ class BLEManager: NSObject, CBPeripheralDelegate, MqttClientProxyManagerDelegate
 						}
 						hopNodes.append(destinationHop)
 						/// Add the destination node to the end of the route towards string and the beginning of teh route back string
```

Comment:
In general this looks good to me. 

One thing is that if `hop_start` is not set in the traceroute response (due to nodes with firmware <2.3 in the route), `routeBack` might be incomplete, since we can't add unknown nodes and SNR in this case (because we don't know how many missing hops there were). I think it would be best to skip showing the route back in this case, like what is done for [Android](https://github.com/meshtastic/Meshtastic-Android/blob/b668a21b68465c13d15ca224df785c86ebe4f3ad/app/src/main/java/com/geeksville/mesh/model/RouteDiscovery.kt#L17) and the [CLI](https://github.com/meshtastic/python/blob/646aa981d5fc191f1900da830615dec91cc77403/meshtastic/mesh_interface.py#L584).

Reply:
I've put a conditional around the routeBack logic. This should match the logic seen in the Andriod app.